src/renderer/components/table/table.tsx

```diff
@@ -92,7 +91,7 @@ export class Table extends React.Component<TableProps> {
     const { sortable, tableId } = this.props;
 
     if (sortable && !tableId) {
-      console.error("[Table]: sorted table requires props.tableId to be specified");
+      throw new Error("Table must have props.tableId if props.sortable is specified");
```

Comment:
Isn't this a breaking change?

Reply:
Yes I guess so, I did this so that it is in line with the other checks (like in `ItemListLayout`.

Reply:
I don't think that this will break any extension developers though...

Reply:
so previously if extension has used sortable table and not given any table id, sorting has not worked and end user has seen the error in console, now instead of displaying error in console, end user will see error notification?

Reply:
> now instead of displaying error in console, end user will see error notification?

I thought they will see error/crash page... that's why I was worried about breaking change.